hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeRequest.java

```diff
@@ -95,13 +100,18 @@ protected VolumeList addVolumeToOwnerList(VolumeList volumeList,
     }
 
     List<String> prevVolList = new ArrayList<>();
+    long objectID = 100;
     if (volumeList != null) {
       prevVolList.addAll(volumeList.getVolumeNamesList());
+      objectID = volumeList.getObjectID();
```

Comment:
Here we always use the objectID of the new volumeList to replace the existing one on line 113, is this expected?

Reply:
Yes, for some reason, when we replace a VolumeList, we do it like this. We read the current list, remove or add a new volume, create a new object and then write it back. This is reality nothing but a small change in the list of volumes. So I keep the same object ID as if Object is same.